code/datums/diseases/advance/symptoms/alcohol.dm

```diff
@@ -35,7 +35,7 @@
 	var/mob/living/carbon/M = A.affected_mob
 	var/list/warningstrings = list()
 	switch(A.stage + severity)
-		if(4 to 5)
+		if(-5 to 5)
```

Comment:
This is equal to default and doesn't need arbitrary -5 value to catch all other cases

Reply:
It doesn't need to be `-5` and I'd rather use `-INFINITY`, but unfortunately, it won't compile with that.
As I've explained in the PR it needs to include the negative values as well.

Reply:
Isn't there a NEGATIVE_INFINITY define

Reply:
`-INFINITY` is a thing. that's weird.

Reply:
Default (in dm else) will catch any value from -infinity to 5